vendor/bat-native-ads/src/bat/ads/internal/ads/search_result_ad_test.cc

```diff
@@ -30,62 +32,76 @@ class BatAdsSearchResultAdIntegrationTest : public UnitTestBase {
     UnitTestBase::SetUpForTesting(/* is_integration_test */ true);
 
     ForcePermissionRules();
+
+    // Need to trigger several search result ad events.
+    privacy::SetUnblindedTokens(20);
```

Comment:
Would be good to set this to the required amount

Reply:
set to 11